kitsune/settings.py

```diff
@@ -1337,8 +1337,6 @@ def filter_exceptions(event, hint):
 
 USER_INACTIVITY_DAYS = config("USER_INACTIVITY_DAYS", default=1095, cast=int)
 
-if DEV:
-    GOOGLE_APPLICATION_CREDENTIALS = config("GOOGLE_APPLICATION_CREDENTIALS", default="")
```

Comment:
`GOOGLE_APPLICATION_CREDENTIALS` is only ever needed for local development. It's never used within the `dev`, `stage`, or `prod` environments.